source/includes/fact-environments-atlas-support-no-free-or-m10.rst

```diff
@@ -1,4 +1,4 @@
 .. note::
```

Comment:
Nonblocking Suggestion:

This is not in the scope of your project, but if you apply a two space indent to the `include` directive used to call this RST file, it'll render as a child of the lists item instead of breaking the list into two separate lists.

Bonus: Since the robots work from simplified HTML, I suspect that'll help their comprehension.

Reply:
spoke to Kenneth over slack, ended up not implementing